index.js

```diff
@@ -1509,32 +1511,37 @@ export async function createJavaBom(path, options) {
       options.installDeps &&
       !["scala", "sbt"].includes(options.projectType)
     ) {
-      const gradleCmd = getGradleCommand(path, null);
-      const defaultDepTaskArgs = ["-q", "--console", "plain", "--build-cache"];
+      let gradleCmd = getGradleCommand(path, null);
+      const defaultDepTaskArgs = ["--console", "plain", "--build-cache"];
       allProjects.push(parentComponent);
       let depTaskWithArgs = ["dependencies"];
       if (process.env.GRADLE_DEPENDENCY_TASK) {
         depTaskWithArgs = process.env.GRADLE_DEPENDENCY_TASK.split(" ");
       }
-      for (const sp of allProjects) {
-        let gradleDepArgs = [
-          sp.purl === parentComponent.purl
-            ? depTaskWithArgs[0]
-            : `:${sp.name}:${depTaskWithArgs[0]}`,
-        ];
-        gradleDepArgs = gradleDepArgs
+      let gradleDepArgs = [];
+      
+      gradleDepArgs = gradleDepArgs
           .concat(depTaskWithArgs.slice(1))
           .concat(defaultDepTaskArgs);
-        // Support custom GRADLE_ARGS such as --configuration runtimeClassPath (used for all tasks)
-        if (process.env.GRADLE_ARGS) {
-          const addArgs = process.env.GRADLE_ARGS.split(" ");
-          gradleDepArgs = gradleDepArgs.concat(addArgs);
-        }
-        // gradle args only for the dependencies task
-        if (process.env.GRADLE_ARGS_DEPENDENCIES) {
-          const addArgs = process.env.GRADLE_ARGS_DEPENDENCIES.split(" ");
-          gradleDepArgs = gradleDepArgs.concat(addArgs);
+      // Support custom GRADLE_ARGS such as --configuration runtimeClassPath (used for all tasks)
+      if (process.env.GRADLE_ARGS) {
+        const addArgs = process.env.GRADLE_ARGS.split(" ");
+        gradleDepArgs = gradleDepArgs.concat(addArgs);
+      }
+      // gradle args only for the dependencies task
+      if (process.env.GRADLE_ARGS_DEPENDENCIES) {
+        const addArgs = process.env.GRADLE_ARGS_DEPENDENCIES.split(" ");
+        gradleDepArgs = gradleDepArgs.concat(addArgs);
+      }
+
+      if (process.env.GRADLE_MULTI_THREADED) {
```

Comment:
Is the idea to retain both the parallel and non-parallel mode for sometime before making parallel mode the default?

Reply:
Correct! Think a slow rollout would be better to ensure that the new method doesn't have any problems. Think it'd be quite useful to leave the single threaded way as is for testing and benchmarking purposes too.